src/ECDSASignatureChecker.sol

```diff
@@ -0,0 +1,170 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import {ECDSARegistryCoordinator} from "./ECDSARegistryCoordinator.sol";
+import {ECDSAStakeRegistry, IDelegationManager} from "./ECDSAStakeRegistry.sol";
+import {EIP1271SignatureUtils} from "eigenlayer-contracts/src/contracts/libraries/EIP1271SignatureUtils.sol";
+
+import {BitmapUtils} from "./libraries/BitmapUtils.sol";
+
+/**
+ * @title Used for checking BLS aggregate signatures from the operators of a `BLSRegistry`.
+ * @author Layr Labs, Inc.
+ * @notice Terms of Service: https://docs.eigenlayer.xyz/overview/terms-of-service
+ * @notice This is the contract for checking the validity of aggregate operator signatures.
+ */
+contract BLSSignatureChecker {    
```

Comment:
Update contract name. also ctrl-f "aggregate" and "bls" and remove any reference.

Reply:
Yes this would need a lot of cleanup w/r/t naming and comments, in particular